### PR TITLE
Prometheus: remove cue definition

### PR DESCRIPTION
**Pull request by ryantxu**
The cue file is slowing down efforts to cleanup the prometheus query type and support scopes concept.  Although the full replacement for a typed query is not yet ready (see https://github.com/grafana/grafana-plugin-sdk-go/pull/897) this PR removes the cue file and works with an easier model.

This has two copies of the same cue file... so obviously something is 🙃 -- rather than fix is, lets just remove it.

Short term, the existing cue files from git history can continue to be used in the required gen pipelines.  Longer term this will be replaced with a JSON schema approach.

NOTE: The  PrometheuQueryProperties model is used in https://github.com/grafana/grafana/pull/82391

### CODE REVIEW

File: pkg/tsdb/prometheus/models/query.go [hunk 1]
```diff
@@ -52,11 +52,12 @@ const (
...
 
 type QueryModel struct {
-	dataquery.PrometheusDataQuery
+	kinds.PrometheusDataQuery
+	RefId string `json:"refId,omitempty"`
+
 	// The following properties may be part of the request payload, however they are not saved in panel JSON
 	// Timezone offset to align start & end time on backend
 	UtcOffsetSec   int64  `json:"utcOffsetSec,omitempty"`
-	LegendFormat   string `json:"legendFormat,omitempty"`
```
Comment (ryantxu) [inline pkg/tsdb/prometheus/models/query.go:59]: ^^ legend is already in `kinds.PrometheusDataQuery` no need to redeclare it

File: pkg/tsdb/prometheus/models/query.go [hunk 2]
```diff
@@ -52,11 +52,12 @@ const (
...
 	Interval       string `json:"interval,omitempty"`
 	IntervalMs     int64  `json:"intervalMs,omitempty"`
 	IntervalFactor int64  `json:"intervalFactor,omitempty"`
```
Comment (ryantxu) [inline pkg/tsdb/prometheus/models/query.go:63]: ^^ ???
this exists in the cue model -- but with a float value 🤷🏻 
https://github.com/grafana/grafana/pull/83808/files#diff-fc88a94ee0629cc8c2b03141bf301f3719a22b57c8332fdcd9c24af15aa4d411R44

**Review by kylebrandt — APPROVED** [review]
BE looks good, @itsmylife / @bohandley can you check FE?

**Comment by itsmylife** [comment]
I'll take a look at FE today. Some stuff is missing I guess but not a big deal. I will point them out.

**Comment by itsmylife** [comment]
I fixed a couple of issues. I'll run a few more tests. 
@fabrizio-grafana I saw you did some changes to decouple tempo. I saw [these files](https://github.com/grafana/grafana/tree/522519f671c2c3affae3ed4208b2cf24406eb68f/public/app/plugins/datasource/tempo/_importedDependencies/datasources/prometheus). 
Could you please check this PR too? You might want to update things in tempo.

File: packages/grafana-prometheus/src/dataquery.ts [hunk 3]
```diff
@@ -1,21 +1,11 @@
...
-//
-// Generated by:
-//     public/app/plugins/gen.go
-// Using jennies:
-//     TSTypesJenny
-//     PluginTSTypesJenny
-//
-// Run 'make gen-cue' from repository root to regenerate.
-
 import * as common from '@grafana/schema';
```
Comment (itsmylife) [inline packages/grafana-prometheus/src/dataquery.ts:1]: Should we keep this?
I think it is necessary "for now", right?

**Review by itsmylife — APPROVED** [review]
LGTM
QueryModel is a bit confusing though but I think for starters it is fine.

**Pull request merged**